spec/tictactoe/ruby_spec.rb

```diff
@@ -1,6 +1,24 @@
-RSpec.describe Tictactoe do
+require "./lib/tictactoe/game"
+RSpec.describe Tictactoe::Game, Tictactoe::Board do
 
-  it "should add 1 to 1" do
-    expect(1+1).to eq(2)
+  context "when both players play" do
+    xit "should signal a draw when there is no winner" do
+
+      game_positions = Array.new(9,"-")
```

Comment:
To me, this is an implementation detail that we don't need to know about the Board class. In acceptance or unit tests, we generally want to define the `behaviors` or `interactions` between objects rather than how you want the classes to be implemented.

To think of it another way, let's say when you start implementing the Board class, you decide to use a database backend to store the game positions instead of an array. You will then be forced to change this test. On the other hand, if you leave the implementation detail of how game positions are stored to Board class, you will not have to change anything here. Essentially, you would make this test more robust and less prone to implementation changes.